### ISSUE TITLE
add link to reference documents

### ISSUE THREAD

**Issue by greg0ire** [ju1ius/sonata-frontend-rfc#1]
Sonata contributors are more backend people, so it would be great to point them to resources needed to understand BEM. https://en.bem.info/method/ , for instance ? Also, explaining why BEM is good and we need to implement it would be great.

**Comment by ju1ius** [comment]
Yep.
Need to noodle on this a bit though, since BEM is a complete methodology, and IMO only the naming convention part will be useful in our case. In this regard, the reference you mentioned could be a bit confusing, making things sound more complex than they actually are.

Another quite important thing would be to mention WAI-ARIA standards as they don't seem to be well-known.

**Comment by greg0ire** [comment]
Yes, this seems to be the maun reference, but maybe there are more understandable blog posts , that may be restricted to what we really need. Regarding WAI-ARIA, I barely know what you are talking about so a mention would definitely be a plus.

**Comment by ju1ius** [comment]
I'll try to find some simplified descriptions online.

[WAI-ARIA](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA) is the [w3c standard](https://www.w3.org/TR/wai-aria-1.1/) for accessibility.
